src/png.imageio/pngoutput.cpp

```diff
@@ -165,10 +170,26 @@ PNGOutput::open(const std::string& name, const ImageSpec& userspec,
         png_set_compression_strategy(m_png, Z_RLE);
     } else if (Strutil::iequals(compression, "fixed")) {
         png_set_compression_strategy(m_png, Z_FIXED);
+    } else if (Strutil::iequals(compression, "pngfast")) {
+        png_set_compression_strategy(m_png, Z_DEFAULT_STRATEGY);
+        png_set_compression_level(m_png, Z_BEST_SPEED);
+    } else if (Strutil::iequals(compression, "none")) {
+        png_set_compression_strategy(m_png, Z_NO_COMPRESSION);
+        png_set_compression_level(m_png, 0);
+        print("compression zero\n");
```

Comment:
Unnecessary `print` I think.

Reply:
oops, you are right, I was using that to debug